secure/storage/vault/src/lib.rs

```diff
@@ -724,7 +793,7 @@ struct ExportKey {
 /// }
 #[derive(Debug, Deserialize, PartialEq, Serialize)]
 struct ListPoliciesResponse {
-    policies: Vec<String>,
+    pub policies: Vec<String>,
```

Comment:
why setting all these fields to pub?

Reply:
Aah, thanks David! This was a legacy thing from an old approach I tried last week! Fixed 😄